modules/configuring-resources-managed-components.adoc

```diff
@@ -0,0 +1,143 @@
+:_content-type: PROCEDURE
+[id="configuring-resources-managed-components"]
+= Configuring resources for managed components on {ocp}
+
+You can manually adjust the resources on {productname-ocp} for the following components that have running pods:
+
+* `quay`
+* `clair`
+* `mirroring`
+* `clairpostgres` 
+* `postgres`
+
+This feature allows users to run smaller test clusters, or to request more resources upfront in order to avoid partially degraded `Quay` pods. Limitations and requests can be set in accordance with link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes[Kubernetes resource units].
+
+You can configure resource requests on the {ocp} UI, or by directly by updating the `QuayRegistry` YAML.
+
+[IMPORTANT]
+====
+The default values set for these components are the suggested values. Setting resource requests too high or too low might lead to inefficient resource utilization, or performance degradation, respectively. 
+====
+
+[id="configuring-resources-ocp-ui"]
+== Configuring resource requests by using the {ocp} UI
+
+Use the following procedure to configure resources by using the {ocp} UI.
+
+.Procedure
+
+. On the {ocp} developer console, click *Operators* -> *Installed Operators* -> *Red Hat Quay*. 
+
+. Click *QuayRegistry*. 
+
+. Click the name of your registry, for example, *example-registry*.
+
+. Click *YAML*. 
+
+. In the `spec.components` field, you can override the resource of the `quay`, `clair`, `mirroring` `clairpostgres`, and `postgres` resources  by setting values for the `.overrides.resources.limits` and the `overrides.resources.requests` fields. For example:
++
+[source,yaml]
+----
+spec:
+  components:
+    - kind: clair
+      managed: true
+      overrides:
+        resources:
+          limits:
+            cpu: "1"   # Limiting to 1 CPU (equivalent to 1000m or 1000 millicpu)
+            memory: "2Gi"   # Limiting to 2 Gibibytes of memory
+          requests: 
+            cpu: "500m"   # Requesting 500 millicpu or 0.5 CPU
+            memory: "512Mi"  # Requesting 512 Mebibytes of memory
+    - kind: postgres
+      managed: true
+      overrides:
+        resources:
+          limits:
```

Comment:
I tried with this configure, it works, basically this doc is good.  
a small question, here and below components, @stevsmit are you intention to only show "limits" or "requests"? I tried if just write "limits:" (without cpu, memory) this works out "no limitation". just want to make sure this is what you want to express.
another small small suggest, the values for quay is too small (I know it's just for demonstration purpose), a proper value maybe easy for end user. fyi, I record the default value in [PROJQUAY-6742](https://issues.redhat.com/browse/PROJQUAY-6742)